lib/surveyor/models/response_methods.rb

```diff
@@ -46,11 +55,15 @@ def answer_id=(val)
       end
 
       def correct?
-        question.correct_answer.nil? or answer.response_class != 'answer' or (question.correct_answer.id.to_i == answer.id.to_i)
+        question.correct_answer.nil? or
+          answer.response_class != 'answer' or
+          (question.correct_answer.id.to_i == answer.id.to_i)
```

Comment:
```suggestion
        question.correct_answer.nil? ||
          answer.response_class != 'answer' ||
          (question.correct_answer.id.to_i == answer.id.to_i)
```

Reply:
given the difference in `or` we just talked about I was going to leave things be...

Reply:
Probably -- though I assume it has no effect here since none of this are assignments. I think the people who wrote this just liked `or` vs. `||`.